sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -99,6 +99,10 @@ add_sycl_plugin(level_zero
     ${XPTI_LIBS}
 )
 
+if (WIN32)
+ target_link_options(pi_level_zero PUBLIC /DEPENDENTLOADFLAG:0x800)
```

Comment:
I would suggest adding comment what "magic number" 0x800 is (LOAD_LIBRARY_SEARCH_SYSTEM32).